app/script/conversation/ConversationRepository.coffee

```diff
@@ -340,6 +340,8 @@ class z.conversation.ConversationRepository
   @return [Boolean] Is the message marked as read
   ###
   is_message_read: (conversation_id, message_id) =>
+    return false if not conversation_id or not message_id
```

Comment:
I hope nothing. Just adding a safeguard in order to verify my hypothesis that `conversation_et = @get_conversation_by_id conversation_id` in L345 sometimes returns undefined and we should continue to move in the Promise direction here as well with proper error handling.